features/back_buttons/present_buttons.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Working-"back" buttons
+  As a user of the platform
+  I want to have working "back" buttons
+  In order to navigate through the platform easily
+
+  Scenario: Visit all pages
+    Given a user
+    And the user is logged in
+    And a tournament 'MyTournamentus' with 4 teams
+    And a tournament 'Tourney2' with 2 users
+    And the Spielplan page for 'MyTournamentus' is visited
```

Comment:
This scenario looks a little bloated, why do we need to visit the tournaments page?

Reply:
To create at least one match and a complete schedule.